packages/react/src/components/MultiSelect/MultiSelect.js

```diff
@@ -465,6 +467,11 @@ MultiSelect.propTypes = {
    */
   useTitleInItem: PropTypes.bool,
 
+  /**
+   * Specify value to control items selected
+   */
+  value: PropTypes.array,
```

Comment:
I think this is a good opportunity to unify prop naming with similar components Dropdown and Combobox and refer to this as `selectedItems`

```suggestion
  /**
   * For full control of the selected items
   */
  selectedItems: PropTypes.array,
```

Reply:
Sure, working on it